buildscripts/build-managed.cmd

```diff
@@ -45,7 +45,7 @@ dir /b "%__DotNetCliPath%\sdk"
 call "!VS%__VSProductVersion%COMNTOOLS!\VsDevCmd.bat"
 echo Commencing build of managed components for %__BuildOS%.%__BuildArch%.%__BuildType%
 echo.
-%_msbuildexe% /ConsoleLoggerParameters:ForceNoAlign "%__ProjectDir%\build.proj" %__MSBCleanBuildArgs% %__ExtraMsBuildParams% /p:RepoPath="%__ProjectDir%" /p:RepoLocalBuild="true" /p:RelativeProductBinDir="%__RelativeProductBinDir%" /p:NuPkgRid=win7-x64 /p:ToolchainMilestone=%__ToolchainMilestone% /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=normal;LogFile="%__BuildLog%"
```

Comment:
Actually it was similar to the failed PR testing builds: `error CS2012: Cannot open 'D:\j\workspace\release_windo---c2468dc6\bin\obj\Windows_NT.x64.Release\ILCompiler.DependencyAnalysisFramework\ILCompiler.DependencyAnalysisFramework.dll' for writing -- 'The process cannot access the file 'D:\j\workspace\release_windo---c2468dc6\bin\obj\Windows_NT.x64.Release\ILCompiler.DependencyAnalysisFramework\ILCompiler.DependencyAnalysisFramework.dll' because it is being used by another process.' [D:\j\workspace\release_windo---c2468dc6\src\ILCompiler.DependencyAnalysisFramework\src\ILCompiler.DependencyAnalysisFramework.csproj]`

Have there been parallel build issues before?  I don't think we've changed anything that would affect this, especially not specifically in these packaging changes.